enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -19,86 +19,159 @@ see-also:
 
 ## Release Signoff Checklist
 
-- [ ] Enhancement is `implementable`
-- [ ] Design details are appropriately documented from clear requirements
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
 - [ ] Test plan is defined
 - [ ] Graduation criteria for dev preview, tech preview, GA
-- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
 
 ## Summary
 
-Current disaster recovery scripts were created before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates. This enhancement allows modifications to the DR scripts such that disaster recovery makes use of the presence of CEO.
-
+The Disaster Recovery (DR) scripts were created prior to OCP 4.4 before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd and provisioning etcd dependencies such as TLS certificates. The introduction of CEO obviated the need for some of the disaster recovery scripts such as scaling and TLS certificate generation.
 
+This enhancement allows modifications to the DR scripts such that disaster recovery makes use of the presence of CEO.
 ## Motivation
 
-The immediate need for enhancement of disaster recovery scripts is to allow basic backup and restore operations to work, which are currently failing the automated tests (e2edisruptive tests). 
-
-In addition, the disaster recovery should allow the recovery of a single failed node, scaling up and scaling down (node drain).
+While some of the manual operations needed in maintaining the health of etcd are now handled by `cluster-etcd-operator` (CEO), manual intervention is still needed when there is an unhealthy member, loss of quorum or need to restore to a previous state.
```

Comment:
The motivation of this enhancement is that while the operator handles many aspects of cluster management, it relies on having a running kube control plane to do so.  In cases where no running kube control plane exists and cannot be started due to an etcd failure, it is necessary to have manual action on the part of a cluster-admin to get back to a running control plane.